docs/src/pages/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Introduction
+weight: -100
+---
+
+The [HealthCare.gov](https://www.healthcare.gov/) design system is a set of open-source design and front-end development resources for creating usable, accessible (e.g., WCAG/Section 508 compliant), responsive, and consistent websites and web-based solutions.
+
+The Healthcare.gov design system is built on the CMS Design System and extends it to support additional CSS and React components.
+
+## Getting started
+
+### Developers
+
+Developers and engineers are encouraged to use React which allows for more complex, interactive components, and many of our React components have built-in accessibility optimizations.
+
+* **Recommended option:** Install the `@cmsgov/ds-healthcare-gov` Node package manager (npm) package if your project uses npm for package management.
+
+    <a href="{{root}}/startup/installation/" class="ds-c-button">Install with npm</a>
+
+
+* If you're unfamiliar with npm and package management, [download the code as a zip file](https://github.com/CMSgov/hcgov-design-system/releases/latest). 
+
+### Designers
+
+Design teams are currently using Figma for creating UIs that follow HealthCare.gov's Design System but will be transitioning to using Sketch in the future. 
```

Comment:
Is it worth linking out to the Figma here if its going to be transitioned away? and you need access too i thought. Maybe we can just have something similar to the Mgov homepage where we ask them to reach out to the team?

Reply:
Since there is no timeline for that transition and right now no resource listed for designer I feel it's OK to list Figma until we have something else - but agree we should add a way for folks to get access.